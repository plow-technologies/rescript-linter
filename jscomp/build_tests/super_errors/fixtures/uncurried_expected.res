let apply = (f) => f(. 1)
let z = apply(x => x+1)
