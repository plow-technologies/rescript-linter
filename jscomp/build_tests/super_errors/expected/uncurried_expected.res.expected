
  [1;31mWe've found a bug for you![0m
  [36m/.../fixtures/uncurried_expected.res[0m:[2m2:15-22[0m

  1 [2mâ”‚[0m let apply = (f) => f(. 1)
  [1;31m2[0m [2mâ”‚[0m let z = apply([1;31mx => x+1[0m)
  3 [2mâ”‚[0m 

  This expression is expected to have an uncurried function